### PR TITLE
Add body names; Closes #46

### LINKED ISSUE #46: Body names
Bodies should have names so an instructor can easily label planets (especially in cases where they are intended to represent things). This is also relevant if we follow through on the idea of loading in simulations from real-world system databases.

### PR DESCRIPTION
**Pull request by aaroncameron21**
Adds body name attribute as well as a generator to create a name if none is given in the constructor.

### CODE REVIEW

**Comment by aaroncameron21** [comment]
I want to update body.update() and it's tests before I merge this.

File: src/planetNamer.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        "ZR"
+    ];
+
+}
+
+PlanetNamer.prototype.getName = function() {
+    return this.prefixes[this.random(0,this.prefixes.length)] + this.suffixes[this.random(0,this.suffixes.length)] + "-" + this.codes[this.random(0,this.codes.length)] + this.random(1000,9999);
+};
+
+PlanetNamer.prototype.random = function (min,max) {
```
Comment (benjic) [inline src/planetNamer.js]: Why put this on the prototype and expose it?

File: test/planetNamer.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  describe('getName', function() {
+    it('should return a string', function() {
+      var p = new PlanetNamer();
+      expect(p.getName()).to.be.a('string');
+    });
+  });
+  describe('random', function() {
+    it('should return a number', function() {
+      var p = new PlanetNamer();
+      expect(p.random()).to.be.a('number');
```
Comment (benjic) [inline test/planetNamer.js]: You could expect it to be within the min and max.

File: src/planetNamer.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        "HC",
+        "Z",
+        "ZX",
+        "ZR"
+    ];
+
+}
+
+PlanetNamer.prototype.getName = function() {
+    return this.prefixes[this.random(0,this.prefixes.length)] + this.suffixes[this.random(0,this.suffixes.length)] + "-" + this.codes[this.random(0,this.codes.length)] + this.random(1000,9999);
```
Comment (benjic) [inline src/planetNamer.js]: [Lodash](https://lodash.com/) is a library that gives you utility functions that make this a little cleaner. Check out [_.sample](https://lodash.com/docs#sample).

File: test/planetNamer.js (hunk 2 above)
Comment (aaroncameron21) [inline test/planetNamer.js]: Whats the best way of testing something with randomness included? What if an edge case that breaks the test is one in a million?

File: src/planetNamer.js (hunk 1 above)
Comment (aaroncameron21) [inline src/planetNamer.js]: Are you suggesting having it be a standalone function? i.e.
`function random(min,max) {
    return Math.floor((Math.random() * (max-min)) + min);
};`

**Comment by aaroncameron21** [comment]
Whoops lodash autocorrected to loads in that commit title. Anyway, using that removed the need to have the random function, so @benjic your suggestions have been incorporated.

**Comment by benjic** [comment]
I can dig it!
